ossm/values.yaml

```diff
@@ -76,8 +76,8 @@ csv:
     * [Documentation](https://docs.redhat.com/en/documentation/red_hat_openshift_service_mesh/latest/html/about/ossm-about-openshift-service-mesh)
     * [Bugs](https://issues.redhat.com/projects/OSSM)
   support: Red Hat, Inc.
-  version: 3.0.3
-  replacesVersion: 3.0.2
+  version: 3.1.0
```

Comment:
I forgot about `deployment.annotations` in this file. We need to:

- update v1_24_6 images with GA images
- add v1_26.2 placeholders (those will be replaced in konflux repo)